pandas/tests/indexes/interval/test_indexing.py

```diff
@@ -424,6 +424,18 @@ def test_get_indexer_interval_index(self, box):
         expected = np.array([-1, -1, -1], dtype=np.intp)
         tm.assert_numpy_array_equal(actual, expected)
 
+    def test_get_indexer_read_only(self):
+        idx = interval_range(start=0, end=5)
+        arr = np.array([1, 2])
+        arr.flags.writeable = False
+        result = idx.get_indexer(arr)
+        expected = np.array([0, 1])
+        tm.assert_numpy_array_equal(result, expected)
+
+        result = idx.get_indexer_non_unique(arr)[0]
+        expected = np.array([0, 1])
+        tm.assert_numpy_array_equal(result, expected, check_dtype=False)
```

Comment:
Hmm what's going on with the dtype? Wouldn't expect that issue

Reply:
Windows/non windows stuff and since the dtype really doesnt matter here, I just ignored it

Reply:
Maybe you can explicitly specify the type of the np.array call instead?

Reply:
This would kill the 32 bit build... Open for suggestions but dtype really doesn't matter